modules/installation-cluster-quickly-extend-workers.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * installing/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+
+[id="installation-cluster-quickly-extend-workers_{context}"]
+= Installing a cluster to quickly extend workers to edge compute pools
+
+You can set the installation program to automatically create public and private subnets for each AWS Local Zone that you defined in your installation configuration file. By using this installation program in this way, you can quickly install a {product-title} cluster with extended worker nodes that operate in AWS Local Zones locations. 
```

Comment:
@mtulio 

Automatic subnets creation by the installation program is still correct?

Reply:
Yep, the installer will create it.